### PR TITLE
Ios11 jailbreak

### PR DESCRIPTION
**Pull request by Memphiz**
this makes kodi compatible with iOS 11 jailbroken devices (Electra jailbreak)

### CODE REVIEW

File: tools/depends/native/gen_entitlements-native/ios11_entitlements.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Rechi) [inline tools/depends/native/gen_entitlements-native/ios11_entitlements.xml]: this is target not native

Comment (Memphiz) [reply tools/depends/native/gen_entitlements-native/ios11_entitlements.xml]: A target file of course. Not sure if I share the separation you are after here ;) Imo the signing process runs on native so it’s native ;)

Comment (wsnipex) [reply tools/depends/native/gen_entitlements-native/ios11_entitlements.xml]: if this xml file is dependent on target, i.e. differs between IOS versions, it should be a target

**Comment by Memphiz** [comment]
Updated - moved entitlements to target

**Comment by Memphiz** [comment]
Anything else missing?

**Comment by MartijnKaijser** [comment]
@Memphiz please merge when you feel it's ready

**Pull request merged**